playbooks/roles/cluster/tasks/cluster_create_generate_inventory.yml

```diff
@@ -1,60 +1,68 @@
-    cli_containerized: False
-    scf_retval_openshift_cert: "/tmp/{{ oo_clusterid }}/certs/wildcard.{{ oo_clusterid }}.{{ domain }}.crt"
-    scf_retval_chain_cert: "/tmp/{{ oo_clusterid }}/certs/rootCA.pem"
-    scf_retval_openshift_key: "/tmp/{{ oo_clusterid }}/certs/wildcard.{{ oo_clusterid }}.{{ domain }}.key"
-    certfile: "{{ scf_retval_openshift_cert }}"
-    scf_retval_router_cert: "/tmp/{{ oo_clusterid }}/certs/wildcard.apps.{{ oo_clusterid }}.{{ domain }}.crt"
-    scr_retval_router_key: "/tmp/{{ oo_clusterid }}/certs/wildcard.apps.{{ oo_clusterid }}.{{ domain }}.key"
-    g_play_dns_role_path: "/tmp/oa_dns_hook"
-  tasks:
-  - name: set AWS credentials
-    set_fact:
-      g_aws_cloud_provider_access_key_id: "{{ aws_access_key }}"
-      g_aws_cloud_provider_secret_access_key: "{{ aws_secret_key }}"
- 
-  - name: Make dns directory
-    file:
-      path: "/tmp/oa_dns_hook/tasks"
-      recurse: yes
-      state: directory    
-
-  - name: Get DNS task
-    get_url:
-      url: https://raw.githubusercontent.com/StevenTobin/openshift-tools/integreatly_cluster_create/ansible/roles/oa_aws_provisioning_dns_hook/tasks/main.yml
-      dest: "/tmp/oa_dns_hook/tasks/main.yml"   
-      force: yes
-
-  - name: create directory for user under /tmp
-    file:
-      path: "/tmp/{{ lookup('env', 'USER') }}/{{ oo_clusterid }}"
-      recurse: yes
-      state: directory
- 
-  - name: set logging flag
-    set_fact:
-        flag_provision_logging: "{% if survey_provision_logging=='true' %}true{% elif survey_provision_logging=='false'%}false{% else %}false{% endif %}"
-
-  - name: create the provisioning_vars.yml
-    template:
-      src: integreatly-inventory.yml.j2
-      dest: "/tmp/{{ oo_clusterid }}/{{ oo_clusterid }}_inventory.yml"
-
-  - name: Create provisioning inventory in tower
-    shell: tower-cli inventory create -n {{ oo_clusterid }}_provisioning_vars --organization secret
-
-  - name: Create cluster inventory in tower
-    shell: tower-cli inventory create -n {{ oo_clusterid }} --organization secret
-
-  - name: Generate <cluster> inventory in tower
-    shell: "awx-manage inventory_import --source=/tmp/{{ oo_clusterid }}/{{ oo_clusterid }}_inventory.yml --inventory-name={{ oo_clusterid }} --overwrite"
-
-  - name: Generate Provisioning_inventory in tower
-    shell: "awx-manage inventory_import --source=/tmp/{{ oo_clusterid }}/{{ oo_clusterid }}_inventory.yml --inventory-name=Provisioning_Inventory --overwrite"
-
-  
\ No newline at end of file
+- name: set AWS credentials
+  set_fact:
+    g_aws_cloud_provider_access_key_id: "{{ lookup('env', 'AWS_ACCESS_KEY_ID' ) }}"
+    g_aws_cloud_provider_secret_access_key: "{{ lookup('env', 'AWS_SECRET_ACCESS_KEY' ) }}"
+    aws_access_key: "{{ lookup('env', 'AWS_ACCESS_KEY_ID' ) }}"
+    aws_secret_key: "{{ lookup('env', 'AWS_SECRET_ACCESS_KEY' ) }}"
+
+- name: Search for tagged AMI
+  ec2_ami_facts:
+    aws_access_key: "{{ aws_access_key }}"
+    aws_secret_key: "{{ aws_secret_key }}"
```

Comment:
Could these be set to:
```
aws_access_key: "{{ g_aws_cloud_provider_access_key_id }}"
aws_secret_key: "{{ g_aws_cloud_provider_secret_access_key }}"
```
We could maybe remove the need for maintaining two sets of AWS cred vars that use the same values

Reply:
> Seems to fail on bootstrap:
> 
> ```
> ansible-playbook -i ../integreatly_dev/inventories/hosts playbooks/bootstrap_cluster.yml --ask-vault-pass
> 
> TASK [cluster : Copy the tower_github_scm key to server.] ****************************************************************************************************************************************************************************************************
> fatal: [127.0.0.1]: FAILED! => {"changed": false, "checksum": "f174f3bdbd6f5c998782411a4cd1b7f360fc48f1", "msg": "Destination directory /home/awx/.ssh does not exist"}
> ```
> Its expecting an awx user to exist on my localhost

Nevermind - need to run `bootstrap_cluster_create.yml` not `bootstrap_cluster.yml`

Reply:
Actually I don't think those cloud_provider* variables are used anymore so I can remove them.